packages/harmony/src/assets/icons/Sparkles.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this the same as IconSpecialAccess?

Reply:
https://harmony.audius.co/?path=/docs/icons-utility-icons--documentation

Reply:
yep i renamed it

Reply:
ah dang good catch